tests/conformance/testdata/Dockerfile.leadingarg

```diff
@@ -0,0 +1,4 @@
+ARG VERSION latest
+ARG FOO bar
+FROM busybox:$VERSION
+ENV FOO $FOO
```

Comment:
I'd rather not add this to the conformance tests, it's a bit of a different beast than the imagebuilder.tests.  I would, however, love to see this Dockerfile used in a test in `tests/bud.bats`.

Reply:
will look into it :wink: